src/SayMore/Model/ArchivingHelper.cs

```diff
@@ -261,6 +261,7 @@ private static void AddIMDISession(Session saymoreSession, ArchivingDlgViewModel
 			var files = saymoreSession.GetSessionFilesToArchive(model.GetType());
 			foreach (var file in files)
 			{
+				if (file.EndsWith(Settings.Default.MetadataFileExtension, StringComparison.InvariantCulture)) continue;
```

Comment:
I can see you are matching the style already present in this file but it continues on the same line are ugly. I wouldn't hold this up, but if you highlighted this loop and did code cleanup it would be interesting to see the results.